CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+# How to contribute to pySDC
+
+1. [Pull Requestion recommendations](./docs/contrib/01_pull_requests.md)
+2. [Continuous Integration](./docs/contrib/02_continuous_integration.md)
+3. [Naming Conventions](./docs/contrib/03_naming_conventions.md)
+4. [Custom Implementations](./docs/contrib/04_custom_implementations.md)
+
+Developments on the `pySDC` code use the classical approach of forks and pull requests from Github.
+There is an [extended documentation](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/getting-started/about-collaborative-development-models) on this aspect (that you can skip if you are already used to it). In addition, some recommendations for pull requests are given [here](./docs/contrib/01_pull_requests.md).
+
+Additionnaly, a _few_ rules are set to enforce code readability, consistency and reliability. Some of them are automatically tested with each commit, and summarized in the page on [continuous integration (CI)](./docs/contrib/02_continuous_integration.md).
```

Comment:
There is one excess n and one missing l in "Additionnaly". I am sure there are spellcheckers for VScode, but it also really easy in vim. You can just write `:set spell` to activate spell checking. When there is a word not in vims dictionary, you can easily add it by writing `:spell pySDC` for instance.
Sorry for nitpicking, but I think we should spell check text only files.

Reply:
Done for all file, and you're right ! Quite easy to activate a spell-checking extension in VSCode, and it won't hurt my approximate English writing :sweat_smile: